invocation-count-measurers/src/bench/scala/org/scalameter/examples/ListBenchmark.scala

```diff
@@ -0,0 +1,29 @@
+package org.scalameter.examples
+
+import org.scalameter.api._
+import org.scalameter.execution.invocation.InvocationCountMatcher
+import org.scalameter.execution.invocation.InvocationCountMatcher.{MethodMatcher, ClassMatcher}
+
+
+object ListBenchmark extends PerformanceTest.Microbenchmark {
+  val sizes = Gen.range("size")(300000, 1500000, 300000)
+
+  val lists = for {
+    size <- sizes
+  } yield (0 until size).toList
+
+  performance of "Option" in {
+    measure method "allocations" in {
+      using(lists) config (
+        exec.independentSamples -> 1 // we rather want one JVM instance since this measurer is deterministic
```

Comment:
Could you clarify, because I'm not sure if you mean that I could override `exec.independentSamples` higher (e.g. in the `performance` line) or something else?